### ISSUE TITLE
Different behaviors of tooltip in IE and Chrome

### ISSUE THREAD

**Issue by t-zeche** [OfficeDev/office-ui-fabric-react#10166]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

#### Actual behavior:

![image](https://user-images.githubusercontent.com/53205695/63141501-7c612480-c018-11e9-9cbd-d2ff1d8b02c7.png)
**This only happens in IE**

#### Expected behavior:

![image](https://user-images.githubusercontent.com/53205695/63141458-59367500-c018-11e9-8d1c-6ff854dfa2d2.png)

**Comment by JasonGore** [comment]
Please provide a CodePen as the template asks for. That will help ensure this isn't a version-specific or implementation detail. It also helps confirm we've fixed the specific usage scenario. Thanks!

**Comment by t-zeche** [comment]
Codepen Link: https://codepen.io/zeche/pen/MWgjWzd
It seems codepen can not run in IE.

**Comment by t-zeche** [comment]
I've tried to export code to codepen from this link:https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip
Tooltip doesn't perform properly as well.

**Comment by JasonGore** [comment]
Yes, I can see the differences at https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip:

IE:
![image](https://user-images.githubusercontent.com/26070760/63285303-45824d00-c26a-11e9-94ea-da2fef5d5442.png)

Chrome:
![image](https://user-images.githubusercontent.com/26070760/63285339-5468ff80-c26a-11e9-89b5-5f57ccf5108c.png)


IE:
![image](https://user-images.githubusercontent.com/26070760/63285352-5cc13a80-c26a-11e9-99aa-c313f928e9c5.png)

Chrome:
![image](https://user-images.githubusercontent.com/26070760/63285361-6480df00-c26a-11e9-93dd-cf996897aacb.png)

As an aside, the formatting of the headings seems to vary quite a bit on the website as well. Maybe related?

**Comment by joschect** [comment]
@t-zeche This might have been just an issue with the examples. It looks like @jdhuntington fixed it in #10178

**Comment by JasonGore** [comment]
This issue is a duplicate of #10050. I missed that during triage.

**Comment by t-zeche** [comment]
Thanks!

**Comment by JasonGore** [comment]
If you are happy with the resolution please close the issue. Thanks!
